api/SpeechSynthesisEvent.json

```diff
@@ -201,37 +201,53 @@
           "spec_url": "https://wicg.github.io/speech-api/#dom-speechsynthesisevent-elapsedtime",
           "support": {
             "chrome": {
-              "version_added": "33"
+              "version_added": "33",
+              "notes": "Units are miliseconds. See <a href='https://crbug.com/1253036'>bug 1253036</a>"
             },
             "chrome_android": {
-              "version_added": "33"
+              "version_added": "33",
+              "notes": "Units are miliseconds. See <a href='https://crbug.com/1253036'>bug 1253036</a>"
             },
             "edge": {
-              "version_added": "14"
-            },
-            "firefox": {
-              "version_added": "49"
-            },
+              "version_added": "14",
+              "notes": "Units are miliseconds. See <a href='https://crbug.com/1253036'>bug 1253036</a>"
+            },
+            "firefox": [
+              {
+                "version_added": "95",
+                "notes": "Units are seconds (spec change). See <a href='https://bugzil.la/1732498'>bug 1732498</a>"
+              },
+              {
+              "version_added": "49",
+              "version_removed": "95",
+              "notes": "Units are miliseconds."
+              }
+            ],
             "firefox_android": {
-              "version_added": "62"
+              "version_added": "62",
+              "notes": "From version 95 the units are seconds. Earlier versions return miliseconds. See <a href='https://bugzil.la/1732498'>bug 1732498</a>"
             },
             "ie": {
               "version_added": false
             },
             "opera": {
-              "version_added": "21"
+              "version_added": "21",
+              "notes": "Units are miliseconds. See <a href='https://crbug.com/1253036'>bug 1253036</a>"
             },
             "opera_android": {
               "version_added": false
             },
             "safari": {
-              "version_added": "7"
+              "version_added": "7",
+              "notes": "Units are in miliseconds from at least version 14.1"
```

Comment:
Not sure what to do here. 14.1.2 is the version that is known to do the right thing, but using seconds might have appeared in an earlier version I guess. It can be tested using https://jsfiddle.net/ybe1c6ot/